### ISSUE TITLE
"name" parameter possible for all providers

### ISSUE THREAD

**Issue by flowerncsu** [CentOS-PaaS-SIG/linchpin#536]
In order to be able to correlate topologies with results, we would like to be able to pass a "name" parameter for any provider and see that in the result set. This already works with AWS, but definitely does not work for beaker. I have not tested other providers.

**Comment by herlo** [comment]
Creating a separate issue for the current critical roles, which are: bkr_server, aws_ec2, dummy_node, and os_server. The remaining roles will be tracked here and will be done at a later time.

**Comment by samvarankashyap** [comment]
Currently i see the following config options on distiller .
```
[distiller]
bkr_server = id,url,system,name
dummy_node: hosts
aws_ec2 = instances.id,instances.public_ip,instances.private_ip,instances.public_dns_name,instances.private_dns_name,instances.tags:name
os_server = servers.id,servers.interface_ip,servers.name,servers.private_v4,servers.public_v4
```
However as mentioned in in #542 it needs testing with all the other providers except aws_ec2 .
Providers needs to be tested : 
1. dummy 
2. bkr_server
3. os_server
4. libvirt
@herlo @p3ck  @flowerncsu Any other requirements suggested are appreciated.

**Comment by samvarankashyap** [comment]
Tested for dummy , os_server . 
Found bug with distiller #573 
Fixed in #577 
Needs testing for libvirt

**Comment by herlo** [comment]
fixed by #629
